src/action.ts

```diff
@@ -47,15 +47,16 @@ function osPlatform() {
 }
 
 function pullRequestBranch(dependency: string, workingDir: string): string {
-  let branch = "patcher-updates"
-
-  if (dependency) {
-    branch += `-${dependency}`
-  }
+  let branch = "patcher"
 
   if (workingDir) {
     branch += `-${workingDir}`
   }
+  branch += "-updates"
+
+  if (dependency) {
+    branch += `-${dependency}`
```

Comment:
(question, non-blocking): When we have `dependency` here - What would be a final example of an going branch name for updates?

Reply:
With a folder also: `pathcher-dev-updates-gruntwork-io/terraform-aws-vpc/vpc-app`.

Without a folder: `pathcher-updates-gruntwork-io/terraform-aws-vpc/vpc-app`.

I will add this as a comment!